monai/handlers/classification_saver.py

```diff
@@ -57,12 +57,12 @@ def __init__(
             filename: if `saver=None`, name of the saved CSV file name.
             overwrite: if `saver=None`, whether to overwriting existing file content, if True,
                 will clear the file before saving. otherwise, will apend new content to the file.
-            batch_transform: a callable that is used to transform the
-                ignite.engine.batch into expected format to extract the meta_data dictionary.
-            output_transform: a callable that is used to transform the
-                ignite.engine.output into the form expected model prediction data.
-                The first dimension of this transform's output will be treated as the
-                batch dimension. Each item in the batch will be saved individually.
+            batch_transform: a callable that is used to extract the `meta_data` dictionary of
```

Comment:
my suggestion is:
batch_transform: a callable that is used to transform the output batch achieved from `ignite.engine.state.batch` to extract the `meta_data` dictionary from the input images. The purpose of extracting the `meta_data` dictionary is to get the filenames of the input images in order to store the classification results.

Reply:
updated in the latest commit.
Thanks.